cloud/firestore/variables.tf

```diff
@@ -0,0 +1,71 @@
+variable "name" {
+  type        = string
+  description = "The name of the Firestore database. Must be unique within the project. Use '(default)' for the default database."
+}
+
+variable "location" {
+  type        = string
+  description = "The location of the database. Available locations: https://cloud.google.com/firestore/docs/locations"
+}
+
+variable "database_type" {
+  type        = string
+  description = "The type of the database. Possible values: FIRESTORE_NATIVE, DATASTORE_MODE"
+  default     = "DATASTORE_MODE"
```

Comment:
default should be firestore since firestore is a next gen replacement for datastore